resolwe_bio/processes/import_data/basespace.yml

```diff
@@ -0,0 +1,55 @@
+- slug: basespace-file-import
+  name: BaseSpace File Import
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/utils:1.2.0
+    resources:
+      network: true
+      secrets: true
+  data_name: 'BaseSpace File Import ({{ file_id|default("?") }})'
+  version: 0.0.1
+  type: data:file
+  category: upload
+  persistence: RAW
+  description: >
+    Import file from Illumina BaseSpace.
+  input:
+    - name: file_id
+      label: BaseSpace file ID
+      type: basic:string
+    - name: access_token_secret
+      label: BaseSpace access token
+      type: basic:secret
+      description: >
+        BaseSpace access token secret handle needed to download the file.
+    - name: verbose
+      label: Verbose
+      description: >
+        Print additional information when errors occur.
+      type: basic:boolean
+      default: false
+  output:
+    - name: file
+      label: File
+      type: basic:file
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      {% set SECRET_PATH = '/secrets/' ~ access_token_secret.handle %}
+
+      BASESPACE_OUTPUT=$(basespace_download.py \
+        --file-id {{ file_id }} \
+        --access-token-secret-path {{ SECRET_PATH }} \
+        {% if verbose %} --verbose {% endif %} \
```

Comment:
This seems wrong to me. Making verbose easily available like this via a parameter could expose secrets. Verbose should only be used for local script debugging and should not be available in production.

Reply:
Another way would be to limit the use of the `verbose` parameter to users with specific privileges. Of course this is not currently supported in Resolwe.

Reply:
The only way is then to remove `verbose` input I guess, but that way we lose the ability to debug in production.

Reply:
Removed `verbose` from the inputs.